From 6d9ec1c760e605466d9033b87973cab2170faf36 Mon Sep 17 00:00:00 2001
From: Nikolaus Rath <Nikolaus@rath.org>
Date: Mon, 1 Feb 2016 10:15:47 -0800
Subject: Use 'cython3' instead of 'cython'

In Debian, the 'cython' command is provided by the Python 2.x version
of Python. While both versions work, it's nicer to use the more recent
version.
---
 setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 9df7..d347 100755
--- a/setup.py
+++ b/setup.py
@@ -211,7 +211,7 @@ class build_cython(setuptools.Command):
 
     def run(self):
         try:
-            version = subprocess.check_output(['cython', '--version'],
+            version = subprocess.check_output(['cython3', '--version'],
                                               universal_newlines=True,
                                               stderr=subprocess.STDOUT)
         except OSError:
@@ -221,7 +221,7 @@ class build_cython(setuptools.Command):
         if not hit or LooseVersion(hit.group(1)) < "0.24":
             raise SystemExit('Need Cython 0.24 or newer, found ' + version)
 
-        cmd = ['cython', '-Wextra', '--force', '-3', '--fast-fail',
+        cmd = ['cython3', '-Wextra', '--force', '-3', '--fast-fail',
                '--directive', 'embedsignature=True', '--include-dir',
                os.path.join(basedir, 'Include'), '--verbose' ]
         if DEVELOPER_MODE:
